onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -21,20 +21,17 @@
         "^test_nesterov_momentum",
         "^test_pow_types_float32_uint32",
         "^test_pow_types_float32_uint64",
-        "^test_sequence_insert_at_back_cpu", // numpy_helper currently not supporting loading segments.
-        "^test_sequence_insert_at_front_cpu", // numpy_helper currently not supporting loading segments.
         "^test_gradient_of_add_and_mul",
         "^test_gradient_of_add",
         "^test_batchnorm_example_training_mode",
         "^test_batchnorm_epsilon_training_mode",
+        "^test_MaxPool2d_stride_padding_dilation_cpu", // result approximation error; need to be updated in ONNX
```

Comment:
Skip the test for now because this test data in ONNX needs to be updated: https://github.com/onnx/onnx/pull/3075 @RandySheriffH will take care of this once he has got the build environment. Thanks.